core/src/main/java/tc/oc/pgm/api/setting/SettingValue.java

```diff
@@ -19,9 +19,10 @@ public enum SettingValue {
   DEATH_FRIENDS(
       "death", "friends", DyeColor.GREEN), // Only send death messages involving yourself or friends
 
-  PICKER_AUTO("picker", "auto", DyeColor.ORANGE), // Display after cycle, or with permissions.
+  PICKER_AUTO("picker", "auto", DyeColor.ORANGE), // Display after cycle, or with permissions
   PICKER_ON("picker", "on", DyeColor.GREEN), // Display the picker GUI always
   PICKER_OFF("picker", "off", DyeColor.RED), // Never display the picker GUI
+  PICKER_MANUAL("picker", "manual", DyeColor.WHITE), // Only display the picker GUI if right clicked
```

Comment:
New setting enum options should be added rl.the bottom, otherwise all current settingswill be wrongly modified for players